### ISSUE TITLE
Change logo so it is 50px tall

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#568]
This commit, https://github.com/VTUL/vtechworks/commit/732ef1fce6b03d1c332968e96af3e0c8d2c5a5c9#diff-e8df17e192778cb6acf0c2b61eaacc67, was not made to the dspace/modules/ folder, so it was not copied in the upgrade from DSpace 5 to 6. Please add this change to the modules folder so it the logo in the header will be 50px high, as it is in DSpace 5.

**Comment by keithgee** [comment]
I resolved this small header issue by taking Amber's old commit and applying it DSpace 6, although it gave a new commit #55a5090c7d54191.  

The plan is to soon - but probably after the initial deployment - we will switch to a new theme with a new header and a slightly different logo.

Unfortunately I couldn't get this to work by copying the file to the dspace/modules area, and then making the change there. I think that explains Amber's choice to use this method, as I spent 2-3 hours on this (way too much time for a simple CSS change, especially given our timeline) before I finally just applied Amber's commit and got it to work.

I may try to overlay this in the dspace/modules folder again, if we even need it, when we get the new theme. I'm fairly certain that I cleared my browser cache and the cocoon cache each time I redeployed, but maybe I missed one or something malfunctioned. Also, there are some differences from running maven in the "<src>/dspace" folder (usually sufficient), and from the "<src>" folder (perhaps necessary for this type of change).

This fix has been deployed to DSpace 6 dev.
